src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -233,6 +233,7 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
                     folderProject,
                     resolutionContext,
                     primaryRepositories,
+                    Console,
```

Comment:
Did we end up filing the bug for removing the use of console in the commandline?